src/target/flashstub/rp.c

```diff
@@ -71,7 +71,7 @@ static gpio_qspi_s *const gpio_qspi = (gpio_qspi_s *)RP_GPIO_QSPI_BASE_ADDR;
 
 #define MIN(x, y) (((x) < (y)) ? (x) : (y))
 
-void __attribute__((naked, used, section(".entry"))) rp_flash_write_stub()
+void __attribute__((naked, used, section(".entry"))) rp_flash_write_stub(void)
```

Comment:
If we need to strictly prototype this, it should have the same signature as `rp_flash_write()` - this is using the lack of `void` in the brackets for its original intended usage, which is taking an unknown set of args.

Reply:
If your system doesn't have `hexdump`, or didn't when your build dir was configured, then Meson will have defaulted to using the pre-built .stub files and thus not rebuild them from the flash stub sources

Reply:
It does have `hexdump`, and I'm regularly removing the`build` directory.

Reply:
The other half of the checks for when to do a Flash stub build, is checking if a firmware build is in progress: https://github.com/blackmagic-debug/blackmagic/blob/778d40363d0df4c192f1c9a02b5e2d24821596ae/src/target/flashstub/meson.build#L37-L41

If you are doing `meson setup build` then you are building BMDA only and thus will always bypass the Flash stub builds too, because the host compiler can't build them.